library/core/class.autoloader.php

```diff
@@ -1068,24 +1068,20 @@ public function mapType() {
     }
 
     /**
-     * Fix path with backslash for Windows OS.
+     * Normalize paths
      *
-     * Check to see if the path is not empty and if it is not in a form of drive: then replace any "\" with "/" to avoid
-     * the parse_fn_file bug and it also further check to see if it has only one "/" and the OS is "WIN" then it will prepend "/"
-     * in front of the path so it has the correct file path.
+     * Replaces any "\" with "/" and prepends another slash to a single leading slash. 
      *
      * @param string $path
      * @return string
      */
     function fixBackSlash($path) {
-        if (!empty($path) && !preg_match('`^[a-z]:`i', $path)) {
```

Comment:
This function was originally written to accommodate for windows network paths, but if the files are on a local disc, it will start with a drive letter. See the issue for this #2860 for an example.

And if it is empty, nothing will be replaced anyway, so I dropped the condition.